extensions/wikia/CreateNewWiki/tasks/ConfigureWikiFactory.php

```diff
@@ -124,36 +149,26 @@ public function getVariablesFromDB( $sharedDBW, $wikiFactoryVariables ) {
 	 * @param $wikiFactoryVariables
 	 */
 	private function setVariables( $sharedDBW, $cityId, $wikiFactoryVariablesFromDB, $wikiFactoryVariables ) {
-		wfProfileIn( __METHOD__ );
+		$rows = [];
+
 		foreach ( $wikiFactoryVariables as $variable => $value ) {
-			/**
-			 * first, get id of variable
-			 */
+
 			$cvId = 0;
-			if ( isset($wikiFactoryVariablesFromDB[$variable]) ) {
+			if ( isset( $wikiFactoryVariablesFromDB[$variable] ) ) {
 				$cvId = $wikiFactoryVariablesFromDB[$variable];
 			}
 
-			/**
-			 * then, insert value for wikia
-			 */
-			if ( !empty($cvId) ) {
-				$sharedDBW->insert(
-					"city_variables",
-					[
-						"cv_value" => serialize( $value ),
-						"cv_city_id" => $cityId,
-						"cv_variable_id" => $cvId
-					],
-					__METHOD__
-				);
+			if ( !empty( $cvId ) ) {
+				$rows[] = [
+					"cv_value" => serialize( $value ),
+					"cv_city_id" => $cityId,
+					"cv_variable_id" => $cvId,
+				];
```

Comment:
previously we'd insert every variable one-by-one—we can batch that for performance